packages/container/src/index.tsx

```diff
@@ -8,7 +8,7 @@ export interface ContainerProps {
 
 const Container: React.FC<ContainerProps> = ({
   children,
-  registryKey = 'context',
+  registryKey = 'containers',
```

Comment:
changed the default here, as when i came to actually use it i made the assumption that `containers` was the key :grin: